index.html

```diff
@@ -214,24 +214,29 @@ <h2>Connect</h2>
           </ul>
         </div>
   
-  
-        <div id="message" class="message">
-            <h2>Message Me</h2>
+        
+        <section id="leave_message" class="leave_message">
+            <h2>Leave a Message</h2>
             <p>Leave a message, and I'll get back to you soon.</p>
   
-            <form class="contact-form">
+            <form class="contact-form" name="leave_message">
               <div class="form-group">
-                <input type="yourname" id="yourname" name="name" placeholder="Name">
+                <input type="yourname" id="yourname" name="name" placeholder="Name" required>
               </div>
               <div class="form-group">
-                <input type="email" id="email" name="email" placeholder="Email address">
+                <input type="email" id="email" name="email" placeholder="Email address" required>
               </div>
               <div class="form-group">
-                <textarea id="message" name="message" placeholder="Message"></textarea>
+                <textarea id="message" name="message" placeholder="Message" required></textarea>
               </div>
               <button type="submit">Send</button>
             </form>          
-        </div>  
+        </section>  
+
+        <section id="messages">
+          <h2>Messages</h2>
+          <ul></ul> 
+      </section>
```

Comment:
some parts of the html do not look formatted. if using the vs code editor, can be done using the https://code.visualstudio.com/docs/languages/html#:~:text=To%20improve%20the%20formatting%20of,just%20format%20the%20selected%20text.